apps/meeting/.eslintrc.json

```diff
@@ -0,0 +1,56 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
```

Comment:
The indent of this file is 4 spaces, do we need to update it?

Reply:
Updated to 2 space indentation.

Reply:
It still looks like 4 indent, is this GitHub display issue?
![image](https://user-images.githubusercontent.com/20471438/147609068-194ddc61-336a-4056-a6df-a11c0798d069.png)